src/cmdc_tools/datasets/base.py

```diff
@@ -19,6 +21,78 @@ def put(self, conn, df):
     def put_prep(self, conn, df):
         pass
 
+    def log_covid_source(
```

Comment:
Are we going to have other source loggers? We'll be ingesting non-covid data as well and it would be nice to have sources automatically tied to it.

Maybe we call this `log_source` and then can special case this function on non-covid data?

Reply:
For now I don't want ot try to guess the schema of a non-covid source table, so I'd vote we keep this covid specific. 

Once we understand the new data sources better we can rename this if it is applicable, or write another similar method if it isn't